ONSdigital/dp-dd-frontend-controller
# Integrate the dd api stub

### What

Integrate the dp-dd-api-stub into the dd front-end controller and provide some basic routes that demonstrate fetching data from the API and rendering it.

### How to review

1. Download and build the dd-api-integration branches of this and the dp-frontend-renderer projects. 
2. Run the fronted-router, front-end renderer, and front-end controller.
3. Navigate to `http://localhost:20000/dd` to see the example dd homepage listing the sample datasets. Drill into a dataset to see a basic list of the dimensions in that dataset.

### Who can review

Anyone but me.
